corehq/apps/zapier/tests/test_fields.py

```diff
@@ -3,6 +3,7 @@
 
 from tastypie.resources import Resource
 
+from corehq.apps.app_manager.models import ConditionalCaseUpdate
```

Comment:
Unused import. I noticed a few other places there the class is imported but not used.